### ISSUE TITLE
Export fails against Keil Uvision5

### ISSUE THREAD

**Issue by dhwalters423** [ARMmbed/mbed-os-example-bootloader#36]
This has been discovered during OOB for Mbed OS 5.6 RC2.

*Platform:* NUCLEO_F439ZI
*Target:* ARM
*Host OS:* Windows 10
*Exporter Target:* Keil Uvision 5

When exporting the application mbed-os-example-blinky with the mbed-os-example-bootloader built against the ARM5 compiler, the following error occurs:

```
C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky>mbed export -i uvision5 -m NUCLEO_F439ZI
Using regions in this build:
  Region bootloader size 0x40000, offset 0x8000000
  Region application size 0x1c0000, offset 0x8040000
Scan: .
Scan: FEATURE_LWIP
Traceback (most recent call last):
  File "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\project.py", line 256, in <module>
    main()
  File "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\project.py", line 252, in main
    build_profile=profile, app_config=options.app_config)
  File "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\project.py", line 95, in export
    app_config=app_config)
  File "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\export\__init__.py", line 341, in export_project
    macros=macros)
  File "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\export\__init__.py", line 207, in generate_project_files
    exporter.generate()
  File "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\export\uvision\__init__.py", line 234, in generate
    ctx.update(self.format_flags())
  File "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\export\uvision\__init__.py", line 168, in format_flags
    flags = copy.deepcopy(self.flags)
  File "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\export\exporters.py", line 111, in flags
    for_asm=True)
  File "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\toolchains\arm.py", line 130, in get_compile_options
    opts += ['--via', self.get_inc_file(includes)]
  File "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\toolchains\__init__.py", line 831, in get_inc_file
    include_file = join(self.build_dir, ".includes_%s.txt" % self.inc_md5)
AttributeError: 'ARM_STD' object has no attribute 'inc_md5'
[mbed] ERROR: "C:\Python27\python.exe" returned error code 1.
[mbed] ERROR: Command "C:\Python27\python.exe -u C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky\mbed-os\tools\project.py -i uvision5 -m NUCLEO_F439ZI --source ." in "C:\Users\davwal04\Desktop\Out of box campaigns\mbed OS 5.6\mbed-os-example-blinky"
---
```

Put here for tracking, but could be systemic to Mbed OS

CLOSED - was using oob1 branch rather than oob2

**Comment by MarceloSalazar** [comment]
cc @adbridge

**Comment by 0xc0170** [comment]
cc @theotherjimmy

**Comment by theotherjimmy** [comment]
@dhwalters423 #5093 Resolved this issue, and is on master and 5.6.0-rc2. Please update your mbed-os

**Comment by dhwalters423** [comment]
Apologies, looks like you are right @theotherjimmy. RC2 is working. I'll close this issue.
